src/plugins/dashboard/common/embeddable/embeddable_saved_object_converters.ts

```diff
@@ -49,7 +48,7 @@ export function convertPanelStateToSavedDashboardPanel(
     type: panelState.type,
     gridData: panelState.gridData,
     panelIndex: panelState.explicitInput.id,
-    embeddableConfig: omit(panelState.explicitInput, ['id', 'savedObjectId']),
+    embeddableConfig: omit(panelState.explicitInput, ['id', 'savedObjectId', 'title']),
```

Comment:
nit:
I noticed that `title` was not omitted even though it is copied on a root level of the object. 
This is causing duplicate title in each panel of dashboard SO which caught my eye in new unit tests.